src/tests/messageUtils.ts

```diff
@@ -0,0 +1,280 @@
+import {
+  KiltKeyringPair,
+  KeyringPair,
+  KiltEncryptionKeypair,
+  DidDocument,
+  NewLightDidVerificationKey,
+  KeyRelationship,
+  DidServiceEndpoint,
+  DidKey,
+  DidVerificationKey,
+  DecryptCallback,
+  EncryptCallback } from '@kiltprotocol/types'
+
+import {
+  Did,
+  SignCallback,
+} from '@kiltprotocol/sdk-js'
+import { Crypto } from '@kiltprotocol/utils'
+import {
+  blake2AsU8a,
+  blake2AsHex,
+} from '@polkadot/util-crypto'
+
+
+
+
```

Comment:
Too many whitespace

```suggestion


```

Reply:
It is removed in: [d2627c3](https://github.com/KILTprotocol/kilt-extension-api/pull/18/commits/d2627c3650aa91bf69830e8e877946fd17c12e29). 

I also added a small GitHub action to check the fmt. I think this is useful since the project gets still more features.